react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {html: false, css: false, js: false};
```

Comment:
Here you only need one property in state, it should just hold the `id` of the open item.